client/app/assets/less/ant.less

```diff
@@ -13,6 +13,10 @@
 @import '~antd/lib/radio/style/index';
 @import '~antd/lib/time-picker/style/index';
 @import '~antd/lib/pagination/style/index';
+@import '~antd/lib/table/style/index';
+@import '~antd/lib/popover/style/index';
+@import '~antd/lib/icon/style/index';
+@import '~antd/lib/tag/style/index';
```

Comment:
<img src="https://user-images.githubusercontent.com/486954/51632289-9beb0f80-1f57-11e9-9e3d-8673df53704a.jpg" width="250" />

Reply:
Let's review this when we remove Angular and Bootstrap :smiling_imp: